packages/compass-shell/config/loaders/inline-entry-loader.js

```diff
@@ -0,0 +1,62 @@
+const loaderUtils = require('loader-utils');
```

Comment:
This file here is mostly copying whatever the [workerize-loader](https://github.com/developit/workerize-loader) is doing, removing all parts that are not really relevant to us: it allows to convert an imported to module to an entry chunk so all its dependencies are bundled with it and then returns a stringified version of the code:

```js
// ./a.js
console.log('Hi!');

// ./b.js
import a from 'inline-entry-loader!./a.js'

console.log(typeof a === 'string'); // true
console.log(a) // "console.log('Hi!');"
```